### PR TITLE
Feature #169 - Implement GET method view for the reviews

### LINKED ISSUE #169: SP: 3 Create GET method view for reviews
Part of user story #25

### PR DESCRIPTION
**Pull request by OleksiiDatsiuk**
I have implemented GET method view for the reviews, to be exact:

- Added default pagination class settings to the settings.py
- Implemented ReviewDisplayView view with GET method for retrieving reviews
- Added URL path to work with this view

Closes #169

### CODE REVIEW

File: beauty/api/urls.py [hunk 1]
```diff
@@ -10,4 +10,6 @@
     path('<int:user>/order/<int:id>/',
          CustomUserOrderDetailRUDView.as_view(),
          name='specialist-order-detail'),
+    path('<int:pk>/reviews/get/', ReviewDisplayView.as_view(),
```
Comment (Misha86) [inline beauty/api/urls.py]: need change URL such a way api/v1/reviews/<int:pk>/

File: beauty/api/views.py [hunk 2]
```diff
@@ -1,22 +1,27 @@
...
+import django_filters
 from django.db.models import Q
 from django.shortcuts import redirect
 from django.utils.encoding import force_str
 from django.utils.http import urlsafe_base64_decode
 from rest_framework import status
 from rest_framework.generics import ListCreateAPIView, get_object_or_404
 from rest_framework.generics import GenericAPIView
-from rest_framework.generics import RetrieveUpdateDestroyAPIView
+from rest_framework.generics import RetrieveUpdateDestroyAPIView,\
```
Comment (Misha86) [inline beauty/api/views.py]: if you import more than one class from one module should use such syntactic (class, class, ...)

File: beauty/api/views.py [hunk 3]
```diff
@@ -1,22 +1,27 @@
...
+    RetrieveAPIView
 from rest_framework.response import Response
 from rest_framework.reverse import reverse
+from rest_framework import filters
 
-from .models import CustomUser, Order
+from .models import CustomUser, Order, Review
```
Comment (Misha86) [inline beauty/api/views.py:17]: if you import more than one class from one module should use such syntactic (class, class, ...)

File: beauty/api/views.py [hunk 4]
```diff
@@ -107,3 +112,28 @@ def get_object(self):
         logger.info(f"{obj} was got for the user {user} (id={user.id})")
 
         return obj
+
+
+class ReviewDisplayView(RetrieveAPIView):
+    """Generic API for custom GET method"""
```
Comment (Misha86) [inline beauty/api/views.py:118]: check your changes using flake8

File: beauty/api/views.py [hunk 5]
```diff
@@ -107,3 +112,28 @@ def get_object(self):
...
+
+    queryset = Review.objects
```
Comment (Misha86) [inline beauty/api/views.py]: why queryset = Review.objects is not queryset = Review.objects.all()?

File: beauty/api/views.py [hunk 6]
```diff
@@ -107,3 +112,28 @@ def get_object(self):
...
+    serializer_class = ReviewDisplaySerializer
+    filter_backends = (filters.OrderingFilter,
+                       django_filters.rest_framework.DjangoFilterBackend, )
+    ordering_fields = ("date_of_publication", )
+    filterset_fields = ("to_user", )
+    ordering = ('-date_of_publication', )
+
+    def get(self, request, *args, **kwargs):
```
Comment (Misha86) [inline beauty/api/views.py]: What does this method reload?

**Review by Misha86 — CHANGES_REQUESTED** [review]
check your changes using flake8

File: beauty/api/views.py (hunk 3 above)
Comment (OleksiiDatsiuk) [reply beauty/api/views.py:17]: image
My imports looks exactly like this, not sure what is wrong

Comment (OleksiiDatsiuk) [reply beauty/api/views.py:17]: I've just added one more class to this list of imports, which were predefined before I started writing my tasks

**Comment by OleksiiDatsiuk** [comment]
Still doesn't meet flake8 requirements will fix it later - right after writing the tests. Since I haven't integrated flake8 yet

File: beauty/api/urls.py (hunk 1 above)
Comment (OleksiiDatsiuk) [reply beauty/api/urls.py]: Got it

File: beauty/api/views.py (hunk 2 above)
Comment (OleksiiDatsiuk) [reply beauty/api/views.py]: Got it

File: beauty/api/views.py (hunk 5 above)
Comment (OleksiiDatsiuk) [reply beauty/api/views.py]: Changed it

**Review by Misha86 — APPROVED** [review]

**Pull request merged**